variables.tf

```diff
@@ -40,6 +40,16 @@ variable "ec2_endpoint" {
   description = "Variables to provision an EC2 endpoint to the VPC"
 }
 
+variable "ecr_api_endpoint" {
+  type = object({
+    private_dns_enabled = bool
+    security_group_ids  = list(string)
+    subnet_ids          = list(string)
+  })
+  default     = null
+  description = "Variables to provision an ECR endpoint to the VPC"
+}
```

Comment:
Same about the order...

Reply:
Uuh... Sorry, but I meant the location of the variable declaration itself, not the fields as they are always using the order `type`, `default`, `description`.